### PR TITLE
SAK-40004 - Library Updates

### PR DESCRIPTION
**Pull request by deinok**
This upgrades some libaries, this will help the transition to JDK9.

### CODE REVIEW

File: master/pom.xml [hunk 1]
```diff
@@ -584,7 +584,7 @@
       <dependency>
         <groupId>dom4j</groupId>
         <artifactId>dom4j</artifactId>
-        <version>1.6.1</version>
+        <version>1.6.1-jboss</version>
```
Comment (steveswinsburg) [inline master/pom.xml]: Can this be bumped to 2.1.0? https://dom4j.github.io/

Comment (deinok) [reply master/pom.xml]: groupId have its diferent.
Would try with org.dom4j

Comment (deinok) [reply master/pom.xml]: Fixed at #1290855

**Comment by ern** [comment]
These changes could be combined under 1 PR... Multiple PR's just make it more complex later on.

Please close on of these PR's.

**Comment by deinok** [comment]
Closing in favour of #5655

**Pull request closed without merging**